eth/backend.go

```diff
@@ -355,41 +355,6 @@ func (s *Ethereum) shouldPreserve(block *types.Block) bool {
 	return s.isLocalBlock(block.Header())
 }
 
-// StartMining starts the miner with the given number of CPU threads. If mining
-// is already running, this method adjust the number of threads allowed to use
-// and updates the minimum price required by the transaction pool.
-func (s *Ethereum) StartMining(threads int) error {
-	// Update the thread count within the consensus engine
-	type threaded interface {
-		SetThreads(threads int)
-	}
-	if th, ok := s.engine.(threaded); ok {
-		log.Info("Updated mining threads", "threads", threads)
-		if threads == 0 {
-			threads = -1 // Disable the miner from within
-		}
-		th.SetThreads(threads)
-	}
-	// If the miner was not running, initialize it
-	if !s.core.IsMining() {
-		// Propagate the initial price point to the transaction pool
-		s.lock.RLock()
-		price := s.gasPrice
-		s.lock.RUnlock()
-		s.core.SetGasPrice(price)
```

Comment:
we should check if we are setting the gas price? probably run this code with transactions for sometime and see if it still works? I feel like this and the accepTxs setting should be checked because it influences the Tx propagation.

Reply:
The only other place we set it is in cmd/go-quai/main.go:319 which is when the developerflag is set.

I think that's the only place it makes sense to set.

Reply:
can you just and see if this branch of logic is touched in the normal running, if then we have to keep it. I am more concerned about the acceptTxs store below this, If this code was not being touched at the moment, we can delete this branch of logic

Reply:
Tested this by putting a panic in line 375, and running all slices. Node doesn't hit this path.